src/lab.rs

```diff
@@ -1,15 +1,20 @@
 use num::Float;
 
 use std::ops::{Add, Sub, Mul, Div};
+use std::marker::PhantomData;
 
 use {Alpha, Xyz, Lch, LabHue};
 use {Limited, Mix, Shade, GetHue, FromColor, ComponentWise};
 use {clamp, flt};
+use ::white_point::{WhitePoint, D65};
 
 use tristimulus::{X_N, Y_N, Z_N};
+//
+// pub type Lab<T> = Lab<D65, T>;
+// pub type Laba<T> = Laba<D65, T>;
 
 ///CIE L*a*b* (CIELAB) with an alpha component. See the [`Laba` implementation in `Alpha`](struct.Alpha.html#Laba).
-pub type Laba<T = f32> = Alpha<Lab<T>, T>;
+pub type Laba<Wp, T = f32> = Alpha<Lab<Wp, T>, T>;
```

Comment:
Here's the problem. `Lab` has no default white point, so that's why it has to be specified somewhere. I think we can safely set `D65` as the default everywhere, just to have something. It can be changed, after all.